src/host/_stream.cpp

```diff
@@ -765,6 +765,12 @@ using Microsoft::Console::VirtualTerminal::StateMachine;
                     Status = AdjustCursorPosition(screenInfo, CursorPosition, dwFlags & WC_KEEP_CURSOR_VISIBLE, psScrollY);
                 }
             }
+            // Notify accessibility to read the backspaced character.
+            // See GH:12735, MSFT:31748387
+            if (screenInfo.HasAccessibilityEventing())
+            {
+                screenInfo.NotifyAccessibilityEventing(CursorPosition.X, CursorPosition.Y, CursorPosition.X + 1, CursorPosition.Y);

```

Comment:
This is an inclusive rect. Did you mean for the `Right` parameter to be `CursorPosition.X + 1` because that would make the event cover 2 positions, not 1.

Reply:
simplified to just notify UIA, so resolving this thread since it's no longer valid